client/db/types.go

```diff
@@ -243,12 +243,13 @@ func decodeMatchProof_v0(pushes [][]byte) (*MatchProof, error) {
 
 // OrderProof is information related to order authentication and matching.
 type OrderProof struct {
-	DEXSig []byte
+	DEXSig   []byte
+	Preimage []byte
 }
 
 // Encode encodes the OrderProof to a versioned blob.
 func (p *OrderProof) Encode() []byte {
-	return dbBytes{0}.AddData(p.DEXSig)
+	return dbBytes{0}.AddData(p.DEXSig).AddData(p.Preimage)
```

Comment:
I modified the serialization without versioning. Any old databases need to be deleted.